l10n_it_fatturapa_in/wizard/wizard_import_fatturapa.py

```diff
@@ -395,7 +395,8 @@ def _prepareInvoiceLine(self, credit_account_id, line, wt_found=False):
         if line.Quantita:
             retLine['quantity'] = float(line.Quantita)
         if (
-            line.PrezzoTotale and line.PrezzoUnitario and line.Quantita and
+            float(line.PrezzoTotale) and float(line.PrezzoUnitario) and
+            line.Quantita and float(line.Quantita) and  # Quantita not required
```

Comment:
da dove lo capisco che non è required? hai link a documentazione?

Reply:
https://www.fatturapa.gov.it/export/fatturazione/it/normativa/f-2.htm